cognite/client/data_classes/contextualization.py

```diff
@@ -607,6 +637,32 @@ class PersonalProtectiveEquipmentDetectionParameters(VisionResource, ThresholdPa
     pass
 
 
+@dataclass
+class DialGaugeDetection(VisionResource, ThresholdParameter):
+    min_level: float | None = None
+    max_level: float | None = None
+    dead_angle: float | None = None
+    non_linear_angle: float | None = None
+
+
+@dataclass
+class LevelGaugeDetection(VisionResource, ThresholdParameter):
+    min_level: float | None = None
+    max_level: float | None = None
+
+
+@dataclass
+class DigitalGaugeDetection(VisionResource, ThresholdParameter):
+    comma_position: int | None = None
+    min_num_digits: int | None = None
+    max_num_digits: int | None = None
```

Comment:
Should we write out the whole name for min/max_num_digits? I.e., min_number_of_digits?
Since we changed from comma_pos to comma_position? @miladh

Reply:
No, API doc is not correct actually: https://github.com/cognitedata/context-api/blob/bc4b1b9ca6f777ea5aee1bcee84c51d4d4ed199f/src/context_api/api/vision/schemas.py#L347-L350

Reply:
fix in API docs: https://github.com/cognitedata/service-contracts/pull/2166/commits/65da23d814e1e976023013fe9357bf6e1098c9a1